docs/tutorials/10_pvqd.ipynb

```diff
@@ -0,0 +1,1009 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Projected Variational Quantum Dynamics\n",
+    "\n",
+    "The projected Variational Quantum Dynamics (p-VQD) algorithm is a quantum algorithm for real time evolution. It's a variational algorithm that projects the state at time $t + \\Delta_t$, as calculated with Trotterization, onto a parameterized quantum circuit.\n",
+    "\n",
+    "For a quantum state $|\\phi(\\theta)\\rangle = U(\\theta)|0\\rangle$ constructed by a parameterized quantum circuit $U(\\theta)$ and a Hamiltonian $H$, the update rule can be written as\n",
+    "\n",
+    "$$\n",
+    "\\theta_{n+1} = \\theta_n + \\arg\\min_{\\delta\\theta} 1 - |\\langle\\phi(\\theta_n + \\delta\\theta)|e^{-i\\Delta_t H}|\\phi(\\theta_n)\\rangle|^2,\n",
+    "$$\n",
+    "\n",
+    "where $e^{-i\\Delta_t H}$ is calculated with a Trotter expansion (using e.g. the [`PauliEvolutionGate`](https://qiskit.org/documentation/stubs/qiskit.circuit.library.PauliEvolutionGate.html) in Qiskit!).\n",
+    "\n",
+    "The following tutorial explores the p-VQD algorithm, which is available as the [`PVQD`](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.PVQD.html) class. For details on the algorithm, see the original paper: [Barison et al. Quantum 5, 512 (2021)](https://quantum-journal.org/papers/q-2021-07-28-512/#)."
```

Comment:
```suggestion
    "where $e^{-i\\Delta_t H}$ is calculated with a Trotter expansion (using e.g. the [PauliEvolutionGate](https://qiskit.org/documentation/stubs/qiskit.circuit.library.PauliEvolutionGate.html) in Qiskit!).\n",
    "\n",
    "The following tutorial explores the p-VQD algorithm, which is available as the [PVQD](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.PVQD.html) class. For details on the algorithm, see the original paper: [Barison et al. Quantum 5, 512 (2021)](https://quantum-journal.org/papers/q-2021-07-28-512/#)."
```
I am guessing the backticks end up confusing things after nbsphinx translates things. Hence remove them - that makes it consistent anyway and they will clearly be links

![image](https://github.com/qiskit-community/qiskit-algorithms/assets/40241007/ff356949-34bd-4357-96a5-36aac53cfca5)

Reply:
yes, this was a reported bug in the sphinx theme, it must have gone unnoticed in the original PR. Thanks for catching it.